presto-product-tests/src/main/java/io/prestosql/tests/hive/acid/TestFullAcidTableRead.java

```diff
@@ -13,36 +13,127 @@
  */
 package io.prestosql.tests.hive.acid;
 
+import io.prestosql.tempto.query.QueryResult;
 import io.prestosql.tests.hive.HiveProductTest;
 import org.testng.SkipException;
+import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
 
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
 import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
 import static io.prestosql.tempto.query.QueryExecutor.query;
 import static io.prestosql.tests.TestGroups.HIVE_TRANSACTIONAL;
 import static io.prestosql.tests.TestGroups.STORAGE_FORMATS;
 import static io.prestosql.tests.utils.QueryExecutors.onHive;
+import static java.util.Locale.ENGLISH;
 
 public class TestFullAcidTableRead
         extends HiveProductTest
 {
-    @Test(groups = {STORAGE_FORMATS, HIVE_TRANSACTIONAL})
-    public void testRead()
+    @Test(groups = {STORAGE_FORMATS, HIVE_TRANSACTIONAL}, dataProvider = "fullAcidTableTypes")
+    public void testSelectFromFullAcidTable(FullAcidTableType type)
     {
         if (getHiveVersionMajor() < 3) {
             throw new SkipException("Presto Hive transactional tables are supported with Hive version 3 or above");
         }
 
-        String tableName = "full_acid_read";
-        onHive().executeQuery("" +
-                "CREATE TABLE " + tableName + "(a bigint)" +
-                "STORED AS ORC TBLPROPERTIES ('transactional'='true')");
+        String tableName = "full_acid_table" + type.name().toLowerCase(ENGLISH);
```

Comment:
You can revert constant change back to `"full_acid_read"`, but would be good to add an undescore.